numba/tests/test_profile_refcount.py

```diff
@@ -0,0 +1,34 @@
+from __future__ import division, absolute_import, print_function
+
+import numba.unittest_support as unittest
```

Comment:
Just a nit, but can you follow PEP 8 for imports order? (stdlib then third-party then current project)
It makes it easier to figure out a module's dependencies.
See https://www.python.org/dev/peps/pep-0008/#imports